storybook/pages/OverviewSettingsPanelPage.qml

```diff
@@ -0,0 +1,33 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+import AppLayouts.Communities.panels 1.0
+
+SplitView {
+    id: root
+    SplitView.fillWidth: true
+
+    OverviewSettingsPanel {
+        SplitView.fillWidth: true
+        SplitView.fillHeight: true
+
+        name: communityEditor.name
+        description: communityEditor.description
+        logoImageData: communityEditor.image
```

Comment:
The `banner` is not needed for `OverviewSettingsPanelPage.qml`. I can add it in the other PR for Edit Community. It will be useful in that page.
WDYT?

Reply:
Added the banner control here: https://github.com/status-im/status-desktop/pull/11341/files#diff-dbbfe4dbe3e5bf93242897fdae83f645dabe99cff310b8a9a569dfd7ffca867bR128-R141